### DISCUSSION TITLE
Proposing research / reading group on personal and collective transformation

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#186] [General]
Let's organize a small regular (biweekly or monthly) reading/research group. We might meet for an hour or two every couple of weeks. I would be willing to take the lead on making this happen.

Subject matter would be things like personal and collective transformation --  its structure, sources and support (aka ontological and cultural evolution).

Questions like:

- what are stages of healthy human development? what are lines of development (e.g. emotional, cognitive, spiritual)?
- What are the collective or cultural analogies of these (and e.g. does it make sense to talk about a Teal or Orange organization or society)? 
- What examples are there of communities and organizations we can learn from?

### Summary

* What: biweekly reading/research group on  
* Why: a small group brings commitment, diversity of knowledge and views etc
* When: date and time tbc with the group once formed
* Where: online
* Who: 3-8 people

### Connection to Life Itself Research Initiative

From 2016 to approx 2019 we ran a regular series of calls and presentations under the umbrella of the [community research initiative](https://next.lifeitself.org/notes/area/research). This would be a continuation of these with a tighter focus.

[the remark this answers is not in the document]
**Reply by theo-cox** [reply]
Would be keen. Probably to avoid overstretch would be inclined to start with once a month initially and do that really properly.

Also interested to critically explore the debate and controversy around the developmental frame, in line with prior chats with John (who I can't seem to tag for some reason) e.g. what do we make of the tensions and controversies and critiques, what other alternative or(/and) complementary lenses exist through which to view transformation, how might we integrate them etc

Also, delving into theories and cases where "inner" transformation links to broader systemic change

**Comment by baouroux** [comment]
Sounds good Theo...also the monthly timing is also perhaps a good rhythm,
at least to begin with.

Since we last spoke, I'm clearer now on how I'd like to focus my writing
project ...
on "developmental democracy", vs. "developmental sensemaking"...presenting
the cognitive scaffolding from the developmental frameworks as a grammar
for discourse in politics and governance.

Plus, your last point on the "inner" transformation reminds me of the
principle in Bohmian dialogue of the going inwards to discover common
ground.

Look forward to it!
John

John Oliver

[the remark this answers is not in the document]
**Reply by rufuspollock** [reply]
OK, great. Will start scheduling probably for jan / feb at this point.

**Comment by rufuspollock** [comment]
cc @msantolini would you be interested especially given research collab with @BoazBF

**Reply by msantolini** [reply]
Absolutely, thanks for the add! Will join for sure, (more easily beginning Feb once I'll be on site at Bergerac).

**Reply by msantolini** [reply]
By the way @rufuspollock the link to the [community research initiative](https://next.lifeitself.org/notebook/area/research) is broken.

**Reply by rufuspollock** [reply]
Fixed. Change notebook to notes

**Comment by Liyubov** [comment]
yes this is great idea. 
which book/paper/reviews you wanted to start with? 

Penny shared some research on role of space for community living, which I am reading and will be reading till Feb I guess. but this is maybe too specific...

**Comment by Alex-Goodall** [comment]
This sounds great. I'd be up for it.

Sources could perhaps include Hanzi's evolution of the integral developmental model and Vervaeke's Ecology of Practices. I believe's he's bringing out a video series on that. Perhaps include the religious dimension, because that's the only way we get something that might, potentially, eventually provide something like a global transformation (e.g. Ken's 'The religion of tomorrow' and Vervaeke's 'the religion that is not a religion'). And to what degree should we move beyond discussion driven by developmental models? For example, what are the implications of embodying a pure idealism perspective (along the lines of Bernado Kastrup's Analytic Idealism - which has captured me pretty strongly), and the left-right brain hemisphere implications of Ian McGilchrist's work  which I believe has profound transformative implications - but it's not clear to me yet how that can be embedded in a practice (but I may well have missed writings on that topic).

Lots of scope!

Alex

**Comment by baouroux** [comment]
Thanks for the great sample of references Alex.... Lots of weaving!

Look forward to how this discussion evolves. Post-developmental,
post-Integral that you evoke (in the sense of course transcend and include)
becoming more mainstream nowadays.

I'm currently launching an Interfaith narratives and arts project, to speak
to the 'union through diversity' lens.... Seeing how evolution as greater
differentiation and inclusion manifests in our future faith practices and
identity.

So very interested to read your references to faith traditions.

All the best,
John

**Comment by lkavanagh30** [comment]
Sounds great. I agree with the comments made already and agree that Jan/Feb
is a good start date.

[the remark this answers is not in the document]
**Reply by BoazBF** [reply]
Thank you dear @rufuspollock for taking a lead on this exciting topic, and I'm very happy to join you all on this journey here. I would be delighted for a 1x/month to start, and see how it goes, whether we specialise/divide forces into more specific subjects.
I am going into silent retreat until end of Jan 2023 now, so will probably miss the first meeting, sorry about that - inch'allah I shall make the following one.
May this be for the benefit of all beings.

[the remark this answers is not in the document]
**Reply by BoazBF** [reply]
I would like to add a few questions I would love to explore:

- How does living in a community serve the integration of higher stages of human development, for individuals and societies?
- What type of development will coommunities offer that can very hardly (or even cannot ) be done individually?
- What forms of development are optimally integrated through relational and communal settings over individual practices?
